README.md

```diff
@@ -1277,5 +1277,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
 - ### **Z**
 
-  - [Zaid Khan](https://github.com/ZaidKhan2002)
   - [Zoheb Alli Khan](https://github.com/ZohebMOPO)
```

Comment:
I think this one should be at the third place

Reply:
hi can you guide me how i can edit the file again? should i create a new PR?